src/main/java/org/example/Extractor.java

```diff
@@ -9,18 +9,57 @@
 import java.util.List;
 
 public class Extractor {
+    
     public List<Chair> getChairs(String url){
-
-        String role, name, email;
         List<Chair> chairs = new ArrayList<>();
-        //todo
+
+        Document document = getDocument(url);
+        if (document == null) return chairs;
+
+        document.select("table.table>tbody>tr").forEach(tr -> {
+            String role, name = "", email = "";
+            if (tr.children().isEmpty() || !tr.children().first().hasText() || !tr.children().first().text().toLowerCase().contains("chair")) {
+                return;
+            }
+            role = tr.child(0).text();
+            Element facultyDetailsElement = tr.child(1);
+            for (Element child : facultyDetailsElement.children()) {
+                if (child.tagName().equals("strong")) {
+                    name = child.select("a").text();
+                } else if (child.tagName().equals("a") && child.attr("abs:href").startsWith("mailto")) {
```

Comment:
You don't need to extract the email addresses based on "mailto" attribute. You can access it through the <a> text.

Reply:
I'm not actually extracting it through the attribute, I'm just making sure that the <a> tag is an email rather than some random link by making sure that the "mailto" attribute is present.